packages/wonder-blocks-color/index.js

```diff
@@ -1,28 +1,33 @@
 // @flow
+import {mix, fade} from "./utils.js";
+
+const offBlack = "#21242c";
+const white = "#ffffff";
+
 const Color = {
     // Product
-    blue: "#1865F2",
-    green: "#00A60E",
-    gold: "#FFB100",
-    red: "#D92916",
+    blue: "#1865f2",
+    green: "#00a60e",
+    gold: "#ffb100",
+    red: "#d92916",
```

Comment:
readability?

Reply:
Yep! And consistency with what's in Zeplin - I was annoyed when I was comparing the colors and having the different casing 😆